.github/workflows/ci_cd.yml

```diff
@@ -13,6 +13,7 @@ on:
     inputs:
       api_branch:
         description: "Branch of the 'ansys-api-acp' repository used during the build."
+        default: 'feat/ply_export_api' # TODO: remove this line before merging
```

Comment:
TODO (maybe just switch to `feat/adaptive_mesh_workflow` for this feature branch)